testsuite/drivers/lkt_toolbox_driver.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from drivers.base_driver import BaseDriver
+
+
+class LktToolboxDriver(BaseDriver):
+    """
+    Custom driver that will automatically run the "lkt_toolbox" utility
+    on the testcase's "test.lkt" file.
+    """
+
+    @property
+    def mandatory_files(self):
+        return ['test.lkt']
+
+    @property
+    def script_and_args(self):
+        return ['lkt_toolbox', 'test.lkt']
+
+    def run(self):
+        derived_env = dict(os.environ)
+
+        if self.valgrind_enabled:
```

Comment:
I believe this is useless: `lkt_toolbox` does not check this environment variable, and we want `lkt_toolbox` to be run under Valgrind anyway in this case. See `run_and_check`'s `memcheck` argument. Also please pass `for_coverage=True`.

Reply:
Thanks for the info. I removed the env variable and added memcheck/for_coverage=True.